media-converter/src/audioconv.rs

```diff
@@ -311,7 +311,7 @@ impl StreamState {
 
             Ok(LoopState::Looping)
         }else{
-            if self.loop_buffers.len() > 0 {
+            if self.loop_buffers.is_empty() {
```

Comment:
This doesn't have iterate

Reply:
This one is missing a negation, right? We want to execute the body if it is _not_ empty.

Reply:
I'm blind - i would have felt terrible if this would have come upstream